ANY API: Add testnets supported devrel operators (#911)
smartcontractkit/documentation: docs/Using Any API/large-responses.md

   - The Chainlink Token address for _Rinkeby_ by calling the [`setChainlinkToken`](/docs/chainlink-framework/#setchainlinktoken) function.
   - The Oracle contract address for _Rinkeby_ by calling the [`setChainlinkOracle`](/docs/chainlink-framework/#setchainlinkoracle) function.

>    - The `jobId`: A specific job for the oracle node to run. In this case, the _data_ is a _bytes_ data type, so you must call a job that calls an API and returns _bytes_. We are going to use a generic **_GET>bytes_** job that can be found [here](/docs/any-api-testnet-oracles/#jobs).

Reviewer: ```suggestion
   - The `jobId`: A specific job for the oracle node to run. In this case, the _data_ is a _bytes_ data type, so you must call a job that calls an API and returns _bytes_. We will be using a generic **_GET>bytes_** job that can be found [here](/docs/any-api-testnet-oracles/#jobs).
```
